docs/gno-infrastructure/setting-up-a-faucet.md

```diff
@@ -0,0 +1,263 @@
+---
+id: setting-up-a-faucet
+---
+
+# Setting up a faucet for your Gno network
+
+In this tutorial, we will cover how to run a local native currency faucet that 
+works seamlessly with a Gno node. Using the faucet, any address can get a hold
+of testnet GNOTs. The faucet used in this tutorial can be found in
+[this repository](https://github.com/gnolang/faucet).
+
+## Prerequisites
+
+- Git
+- Go 1.21+
+- Make (for running Makefiles)
+- `gnoland` & `gnokey` installed
+- A `gnoland` node running
+- A Gno.land keypair generated using [`gnokey`](../gno-tooling/cli/gnokey.md) 
+containing funds for the faucet to serve
+
+## Cloning the repo
+
+To get started with setting up the faucet, visit the 
+[faucet repo](https://github.com/gnolang/faucet) and clone it:
+
+```bash
+git clone git@github.com:gnolang/faucet.git
+```
+
+After going into the cloned folder, you can build out the faucet binary:
+```bash
+make build
+```
+
+We are now ready to configure & run the faucet.
+
+## Configuring the faucet
+
+By running the `generate` command from the faucet binary, you will be able to generate
+a `config.toml` file.
+
+```bash
+./build/faucet generate
+```
+
+In the `config.toml` file, you will be able to configure a few parameters:
+- ChainID of the node to connect to
+- Faucet listener address
```

Comment:
Please expand on this, it's not immediately clear that this is the **serve** address of the faucet that others can use to initiate faucet requests